src/main/kotlin/com/openlattice/graph/Graph.kt

```diff
@@ -297,12 +300,52 @@ class Graph(private val hds: HikariDataSource, private val edm: EdmManager) : Gr
          * | person_ek | 1_assoc_ptId_agg | 2_assoc_pt_id_agg | 1_entity_ptId_agg | 2_entity_ptId_agg | score
          */
 
-        val idColumns = if (linked) {
-            listOf(LINKING_ID.name to EdmPrimitiveTypeKind.Guid)
-        } else {
-            listOf(SELF_ENTITY_SET_ID to EdmPrimitiveTypeKind.Guid, SELF_ENTITY_KEY_ID to EdmPrimitiveTypeKind.Guid)
-        }
 
+        val idColumns = listOf(
+                SELF_ENTITY_SET_ID to EdmPrimitiveTypeKind.Guid,
+                SELF_ENTITY_KEY_ID to EdmPrimitiveTypeKind.Guid)
+
+        val aggregationColumns = filteredRankings.mapIndexed { index, authorizedFilteredRanking ->
+            buildAggregationColumnMap(
+                    index,
+                    authorizedFilteredRanking.associationPropertyTypes,
+                    authorizedFilteredRanking.filteredRanking.associationAggregations,
+                    ASSOC
+            ).map { it.value to authorizedFilteredRanking.associationPropertyTypes[it.key]!!.datatype } +
+                    buildAggregationColumnMap(
+                            index,
+                            authorizedFilteredRanking.entitySetPropertyTypes,
+                            authorizedFilteredRanking.filteredRanking.neighborTypeAggregations,
+                            ENTITY
+                    ).map { it.value to authorizedFilteredRanking.entitySetPropertyTypes[it.key]!!.datatype } +
+                    (associationCountColumnName(index) to EdmPrimitiveTypeKind.Int64 )+
```

Comment:
What's up with the weird whitespace?

Reply:
What do you mean by that?

Reply:
There is a space between Int64 and paranthesis but not so on the next line. There is also no space before the plus, but other plus signs have a space.